### ISSUE TITLE
Question about save and load ckpt

### ISSUE THREAD

**Issue by hunterkun** [tensorpack/tensorpack#1398]
I read the doc 'Save and Load models' and use `load_ckpt_vars` to get variables dict from ckpt file. I found it contains all tensors except missing their slots variables like `BatchNorm/beta/Momentum`.  When I made some changes and save to ckpt file then restore back to finetune, I found the warning `BatchNorm/beta/Momentum is not available in checkpoint` . Although missing these triaing-related variables `*/Momentum` can also finetune, I want to ask whether the random-initialized those slot variables would affect the finetuning Or it is normal to miss those slot variables in finetuning?

**Comment by ppwwyyxx** [comment]
`load_chkpt_vars` returns everything in the checkpoint. So those variables you're interested in do not exist in the checkpoint.

It's ok (and also normal) to fine tune without momentum.

**Comment by hunterkun** [comment]
Thanks for your instant reply. I got it. But I print those variables names and indeed lacks slots variables.
```
reader = NewCheckpointReader(path)
var_names = reader.get_variable_to_shape_map().keys()
```
